### DISCUSSION TITLE
[Windows] Just pipe chafa output into fastfetch?

### DISCUSSION THREAD

**Discussion by Lil-Dank** [fastfetch-cli/fastfetch#628] [Q&A]
Since I dont know how to compile fastfetch with chafa and it seems too difficult for me I thought why not just make it possible to pipe the output from chafa to fastfetch

or maybe pass the chafa executable and make chafa available to use with fastfetch easily.

Or maybe im missing something and there is an easy way to use chafa with fastfetch

**Comment by CarterLi** [comment]
![image](https://github.com/fastfetch-cli/fastfetch/assets/6134068/27ac6503-a37a-46ad-ba43-24417c0af4bc)

**Reply by Lil-Dank** [reply]
When I try this it looks really strange:
![image](https://github.com/fastfetch-cli/fastfetch/assets/128683657/af253658-bca7-4124-980a-68ad2b80aece)
![image](https://github.com/fastfetch-cli/fastfetch/assets/128683657/db2bc9f6-bc16-477c-9ed8-2c9b567078d1)

**Reply by CarterLi** [reply]
Can you send me the image and the generated text file?

**Reply by Lil-Dank** [reply]
![samurai_logo-orig](https://github.com/fastfetch-cli/fastfetch/assets/128683657/abf1b5ba-8f7a-4bdd-baca-ba5d885f0340)
[logo.txt](https://github.com/fastfetch-cli/fastfetch/files/13441440/logo.txt)

Like this?

**Comment by CarterLi** [comment]
[img.txt](https://github.com/fastfetch-cli/fastfetch/files/13441696/img.txt)

Seems that powershell messed the file encodings. This file works for me
